Source/Core/ExperimentalFeatures.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
I'd like to make this comment more in depth to give some guidance on when it's appropriate to use this, see #9639. I can do this one in my documentation sweep

Reply:
@sanjeetsuhag Updated.

@ebogo1 you might also want to look over this comment too to see if this is a good description of the things we discussed in #9639 so we're all on the same page.

Reply:
looks good to me, thanks!